pom.xml

```diff
@@ -308,8 +308,10 @@
                 <module>app-authz-uma-photoz</module>
                 <module>fuse63</module>
                 <module>fuse70</module>
-                <module>user-storage-simple</module>
-                <module>user-storage-jpa</module>
+                <!-- Requires updating to 9.0.1-SNAPSHOT and above -->
```

Comment:
Should already be resolved in master now.

Reply:
I believe it's not: https://github.com/keycloak/keycloak/pull/6795

Reply:
+1, not merged yet. I will try someone from my sub-team to review that, but if anyone of you has free cycles to review as well, it will be nice. More info in the team ML thread "Supported SPIs"

Reply:
@slaskawi fyi. The PR https://github.com/keycloak/keycloak/pull/6795 is merged already. user-storage-* quickstarts should be compilable against latest keycloak master.